knowledge-base/how-to-create-multiselect-checkbox-parameter-editor.md

```diff
@@ -0,0 +1,107 @@
+---
+title: How to Create MultiSelect CheckBox Parameter Editor
+description: Use kendo's CheckBoxGroup widget with multivalue parameters
```

Comment:
You can consider changing it to The article explains how to display a multivalue parameter by using the CheckBoxGroup widget from Kendo UI